_harp/eclipse_lyo/setup-an-oslc-provider-consumer-application.md

```diff
@@ -87,112 +162,107 @@ We require Java EE 6 or higher and JSTL:
 </dependency>
 ```
 
-### Lyo repositories
-
-an OSLC4J project will use Lyo dependencies that we need to declare. Before we
-can do that, we need to add the Lyo repository. After that, we are ready to add the dependencies.
+## JAX-RS implementation dependencies
 
+### for Lyo 4.0.0-SNAPSHOT 
+For Lyo release 4.0.0-SNAPTSHOT, you will need to choose a JAX-RS 2.0 implementation, such as [Jersey](https://jersey.github.io/), [RESTEasy](https://resteasy.github.io/), etc. Below is an example for Jersey.
 ```xml
-  <repositories>
-    <repository>
-      <id>lyo-releases</id>
-      <name>lyo-releases repository</name>
-      <url>https://repo.eclipse.org/content/repositories/lyo-releases/</url>
-    </repository>
-  </repositories>
+<dependency>
+  <groupId>org.glassfish.jersey.core</groupId>
+  <artifactId>jersey-server</artifactId>
+  <version>2.25.1</version>
+</dependency>
+<dependency>
+  <groupId>org.glassfish.jersey.containers</groupId>
+  <artifactId>jersey-container-servlet</artifactId>
+  <version>2.25.1</version>
+</dependency>
 ```
 
-### SLF4J package dependencies
+### for Lyo 2.4.0 and earlier 
+For Lyo release 2.4.0 (and earlier), a Lyo package includes a dependency to the [Apache Wink implementation](https://svn.apache.org/repos/infra/websites/production/wink/content/index.html).
```

Comment:
They moved to Git and then to Attic

Reply:
Anything i need to do about this comment?